app/models/container_project.rb

```diff
@@ -35,6 +35,8 @@ class ContainerProject < ApplicationRecord
   virtual_total :containers_count,  :containers
   virtual_total :images_count,      :container_images
 
+  after_create :raise_creation_event
```

Comment:
In graph refresh batch saving, we do not call these, you can add it in:
https://github.com/Ladas/manageiq-providers-kubernetes/blob/fb3e5d8b1525d54319dcefecd914e1b135f75a1e/app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb#L86

so in similar way as for container_images, we can access persistor.container_projects.created_records.

Reply:
:+1: ManageIQ/manageiq-providers-kubernetes#226